stable/mediawiki/templates/_helpers.tpl

```diff
@@ -15,6 +15,20 @@ We truncate at 63 chars because some Kubernetes name fields are limited to this
 {{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
 
+{{/*
+Create chart name and version as used by the chart label.
+*/}}
+{{- define "mediawiki.chart" -}}
+{{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Return the proper Mediawiki image name
+*/}}
+{{- define "mediawiki.image" -}}
+{{- printf "%s/%s:%s" .Values.image.registry .Values.image.repository .Values.image.tag -}}
```

Comment:
check with @carrodher as he is fixing a corner case with this image functions

Reply:
@carrodher do you see anything weird?

Reply:
maybe you should define like we do in ES and other charts, to be consistent:
https://github.com/bitnami/charts/blob/master/bitnami/elasticsearch/templates/_helpers.tpl#L25-L32

Reply:
I don't want to use "defaults". Defaults should be provided by "values.yaml"